src/ParcelRegistry/Parcel/ValueObjects/VbrCaPaKey.cs

```diff
@@ -0,0 +1,20 @@
+namespace ParcelRegistry.Parcel
+{
+    using System;
+    using Be.Vlaanderen.Basisregisters.AggregateSource;
+    using Be.Vlaanderen.Basisregisters.Generators.Guid;
+    using Be.Vlaanderen.Basisregisters.GrAr.Common;
+    using Newtonsoft.Json;
+
+    public class VbrCaPaKey : StringValueObject<VbrCaPaKey>
+    {
+        private static readonly Guid Namespace = new Guid("28068113-774a-4f6f-8264-9ddb39095bd4");
```

Comment:
@ArneD we have copy-pasted this class. So the Guid is the same as in Legacy. Should it be the same as in legacy?

Reply:
Done.